snmp/tests/test_e2e_core_profiles/test_profile_3com.py

```diff
@@ -32,6 +34,7 @@ def test_e2e_profile_3com(dd_agent_check):
     assert_common_metrics(aggregator, common_tags)
     assert_extend_generic_if(aggregator, common_tags)
     aggregator.assert_all_metrics_covered()
+    aggregator.assert_metrics_using_metadata(get_metadata_metrics(), check_submission_type=True)
```

Comment:
One more thing, could you remove `check_submission_type=True` from  `test_e2e_core_profiles` and `test_e2e_core_vs_python.py` that doesn't need it ?


The `check_submission_type=True` is only needed for python integration tests (e.g. test_profiles.py).


The python integration tests report "submission type", but e2e tests report "in-app" types:
https://docs.datadoghq.com/metrics/types/?tab=count

Example: 
![image](https://github.com/DataDog/integrations-core/assets/49917914/e2477571-066e-47d1-9663-6bad5c605a4c)

Reply:
Done ✅